src/database/index.tsx

```diff
@@ -397,16 +397,25 @@ export const saveFacility = async (
     if (facility.id) {
       const payload = buildUpdatePayload(facility);
       facilityDoc = facilitiesCollection.doc(facility.id);
-      // Only update the facility if the incoming observedAt date is > than the current observedAt date in the existing facility
-      const incomingObservedAt = facility.modelInputs.observedAt;
-      const currentFacilityData = await facilityDoc.get();
-      const currentFacility = buildFacility(scenarioId, currentFacilityData);
-      if (
-        incomingObservedAt &&
-        currentFacility.modelInputs.observedAt &&
-        startOfDay(incomingObservedAt) >=
-          startOfDay(currentFacility.modelInputs.observedAt)
-      ) {
+
```

Comment:
Minor regression I found when testing Multi-scenario stuff.  Not crazy about all of these nested conditionals, but wanted to get this in before today's production deployment